src/orquestra/sdk/_base/abc.py

```diff
@@ -179,6 +185,28 @@ def iter_logs(
         """
         raise NotImplementedError()
 
+    @abstractmethod
+    def list_workflow_runs(
+        self,
+        *,
+        limit: t.Optional[int] = None,
+        prefix: t.Optional[str] = None,
```

Comment:
**question**: you mentioned earlier that we planned to not use the ID prefix. We don't use it in the Python API. Was it intentional to put this param here?

Reply:
done in [a2d40d9](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/3/commits/a2d40d91e2148521c57fc627af37fc24c462d5bd)